src/tabs/failsafe.html

```diff
@@ -134,8 +134,8 @@
                                     </label>
                                 </div>
                                 <div class="number">
-                                    <label> <input type="number" name="gps_rescue_min_dth" min="50" max="1000" /> <span
-                                        i18n="failsafeGpsRescueItemMinDth"></span>
+                                    <label> <input type="number" name="gps_rescue_min_dth" min="20" max="1000" />
+                                        <span i18n="failsafeGpsRescueItemMinDth"></span>
```

Comment:
I suppose this must be done depending on the version and not for all?

Reply:
Thanks. Fixed.